Automattic/jetpack
# VideoPress: Add feedback link to both VideoPress blocks

This PR adds a link to the current VP block as well as the new VP block that is shipped in the standalone VideoPress plugin for users to give us feedback on the product:

Current block:
<img width="290" alt="Screenshot 2022-11-16 at 10 35 12 AM" src="https://user-images.githubusercontent.com/789137/202253963-a1641a8d-02dd-4694-a9e8-e7f635cf65ae.png">

New block:
<img width="347" alt="Screenshot 2022-11-16 at 10 06 51 AM" src="https://user-images.githubusercontent.com/789137/202253965-927aaebf-6fd8-47dc-86ce-24278ca4a1a0.png">

#### Changes proposed in this Pull Request:
* Adds an external link to a Crowdsignal feedback survey from the VP block editor sidebar.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Activate the main Jetpack plugin, and activate Jetpack VideoPress.
* From the post editor, add a video block.
* Open the sidebar, and look for the Feedback link. Clicking the link should take you to the survey.
* Deactivate Jetpack plugin, and activate the standalone Jetpack VideoPress plugin.
* From the post editor, add a video block.
* Open the sidebar, and look for the Feedback link. Clicking the link should take you to the survey.
